proxy/src/main/java/com/velocitypowered/proxy/connection/backend/BackendPlaySessionHandler.java

```diff
@@ -270,6 +272,16 @@ public boolean handle(AvailableCommands commands) {
     return true;
   }
 
+  @Override
+  public boolean handle(ServerPlayerChat packet) {
+    HeaderData chatHeader = packet.getHeaderData();
+    if (chatHeader != null) {
+      ConnectedPlayer who = this.serverConn.getPlayer();
+      who.pushHeader(chatHeader);
```

Comment:
This causes gradle to fail - Is there a library missing for it?

Reply:
It was a push of a WIP set of changes is my understanding, stuff is still a WIP

Reply:
As cat said, this was me saving progress